README.md

```diff
@@ -97,6 +108,39 @@ $ ks apply default
 
 TODO
 
+## Customising the mixin
+
+Kubernetes-mixin allows you to override the selectors used for various jobs,
+to match those used in your Prometheus set.
+
+In a new directory, add a file `mixin.libsonnet`:
+
+```
+local kubernetes = import "kubernetes-mixin/mixin.libsonnet";
+
+kubernetes {
```

Comment:
should this be `kubernetes + {` ?

Reply:
what i've put in is shorthand - they do the same.